oak/proto/oak_api.proto

```diff
@@ -41,4 +41,6 @@ enum ChannelHandle {
   LOGGING = 1;   // Write half of channel
   GRPC_IN = 2;   // Read half of channel
   GRPC_OUT = 3;  // Write half of channel
-}
\ No newline at end of file
+  STORAGE_READ = 4;
```

Comment:
I think `STORAGE_IN` and `STORAGE_OUT` make more sense here, both for consistency, and also so they don't suggest that one is for reading and the other for writing data to storage.

Reply:
Done.